apps/bondy/src/bondy_wamp_protocol.erl

```diff
@@ -101,7 +101,22 @@ format_status(Opt, #wamp_state{} = State) ->
     State#wamp_state{
         auth_context = NewAuthCtxt,
         context = NewCtxt
-    }.
+    };
```

Comment:
Here I realized later that `format_status` is a standard method for `gen_server`. So all these extra definitions are probably irrelevant. I left them because... Well, I wrote them..

Reply:
yes it is.

I would replace this lines with just 

```erlang
format_status(_, undefined) ->
    undefined.
```

Reply:
@alan-ls can you change this as recommended above so I can merge?